packages/devtools_app/lib/src/screens/performance/panes/controls/enhance_tracing/enhance_tracing_controller.dart

```diff
@@ -28,3 +70,31 @@ class EnhanceTracingController extends DisposableController
     super.dispose();
   }
 }
+
+@immutable
+class EnhanceTracingState {
```

Comment:
This class is a data structure, without logic. Should it go to model, to make the library 'controller' being focused on business logic, not on data structures?

Reply:
done 👍